### DISCUSSION TITLE
Is access to dislikes via YouTube API disabled already?

### DISCUSSION THREAD

**Discussion by Leopold702** [Anarios/return-youtube-dislike#337] [Q&A] [answered]
As I understand YouTube API was disabled on December 13th, however even on videos uploaded after that date the extension displays pretty convincing numbers of dislikes. So I wonder, are dislikes we see on the newly uploaded videos collected exclusively from the extension users, or the data somehow still pulled from the YouTube API?

**Comment by Anarios** [comment, marked as answer]
Youtube disabled the api on december 13th. Average rating (hidden html field) was available for a little bit longer.

**Reply by hofq** [reply]
> Youtube disabled the api on december 13th. Average rating (hidden html field) was available for a little bit longer.

If it's Gone now, could you correct the Readme and the FAQ File? The formulation  could confuse some people.
